Gopkg.toml

```diff
@@ -1,5 +1,13 @@
 # Refer to https://golang.github.io/dep/docs/Gopkg.toml.html for details
 
+[[override]]
+  name = "k8s.io/client-go"
+  revision = "56e7a63b5e38b614bb7cb8c49ed26dc991b324c4"
+
+[[override]]
+  name = "istio.io/api"
+  revision = "3094619c84733caef53723bfc96fa63ceb58cd57"
```

Comment:
Kevin mentioned Istio aimed at releasing 1.1 this week, though he was sceptical. There has been more work in this repo's `release-1.1` branch, as well as on the istio/istio repo further down, perhaps good to update them to the closest point possible (ie. today) before release?

Reply:
I don't think that is necessary. We depend only on the logger and the test framework. I'll what for a release and bump then if you have no objections.

Reply:
That's ok.